pavelib/utils/test/suites/bokchoy_suite.py

```diff
@@ -116,19 +145,25 @@ def cmd(self):
         else:
             test_spec = self.test_dir / self.test_spec
 
+        # Skip any additional commands (such as nosetests) if running in
+        # servers only mode
+        if self.serversonly:
+            return ""
+
         # Construct the nosetests command, specifying where to save
         # screenshots and XUnit XML reports
-        cmd = [
-            "DEFAULT_STORE={}".format(self.default_store),
-            "SCREENSHOT_DIR='{}'".format(self.log_dir),
-            "BOK_CHOY_HAR_DIR='{}'".format(self.har_dir),
-            "SELENIUM_DRIVER_LOG_DIR='{}'".format(self.log_dir),
-            "nosetests",
-            test_spec,
-            "--with-xunit",
-            "--xunit-file={}".format(self.xunit_report),
-            "--verbosity={}".format(self.verbosity),
-        ]
+        else:
```

Comment:
This doesn't need to be in an else block because the if branch would have already returned.